### PR TITLE
Update/add create alerts and update alert types content 

### PR DESCRIPTION
**Pull request by imatwawana**
This PR:
- Adds the **Create Alerts** section and index page
- Moves/updates Issue and Metric alert how-to content from Sentry Basics to new configuration pages 
- Creates new **Alert Types** page with remaining/updated content from previous Issue and Metric Alert pages 
- Deletes old Issue Alerts page (content moved to Alert Types page)
- Changes top-level folder name to **Alerts** from **Alerts & Notifications** 

Pages 
- Create Alerts
- Issue Alert Configuration
- Metric Alert Configuration
- Alert Types (formerly Metric Alerts)
- Issue Alerts (deleted)

### CODE REVIEW

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+- Hours: 3, 12, 24
+- Days: 7, 30
+
+## Project-Level Alert Settings
+
+In **[Project] > Settings > Alerts**, you can configure alert email subject templates and digest settings. Sentry users with owner, manager, or admin permissions and above can change these default notification settings.
+
+### Digests
+
+The digests feature only works for issue alert emails (not notifications sent through integrations), and unlike the [action interval](##action-interval), limits the total number of alert emails sent for the project. This project-level setting allows you to control the minimum and maximum delivery intervals for alerts.
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:91]:
```suggestion
The digests feature only works for issue alert emails (not notifications sent through integrations), and unlike the [action interval](#action-interval), limits the total number of alert emails sent for the project. This project-level setting allows you to control the minimum and maximum delivery intervals for alerts.
```

File: src/docs/product/sentry-basics/guides/alert-notifications/notifications.mdx [hunk 2]
```diff
@@ -10,7 +10,7 @@ Manage [your notifications](https://sentry.io/settings/account/notifications/) b
 The "Send Me Alerts" toggle connects you to alerts sent to your teams. If you've toggled "Send Me Alerts" on, that setting serves as your default for each project. You can also fine tune your alerts according to project by choosing between "default," "on," or "off."
 
 ## Workflow Notifications
-Workflow Notifications are emails sent for [issue state](/product/alerts-notifications/notifications/#issue-states) changes. Your default choices for notifications are "Always," "Only On Issues I Subscribe To," and "Never." You can also fine tune your workflow notifications according to project by choosing "Default," "Always," "Only On Issues I Subscribe To," or "Never."
+Workflow Notifications are emails sent for [issue state](/product/alert-notifications/notifications/#issue-states) changes. Your default choices for notifications are "Always," "Only On Issues I Subscribe To," and "Never." You can also fine tune your workflow notifications according to project by choosing "Default," "Always," "Only On Issues I Subscribe To," or "Never."
```
Comment (imatwawana) [inline src/docs/product/sentry-basics/guides/alert-notifications/notifications.mdx:13]:
```suggestion
Workflow Notifications are emails sent for [issue state](/product/alerts/notifications/#issue-states) changes. Your default choices for notifications are "Always," "Only On Issues I Subscribe To," and "Never." You can also fine tune your workflow notifications according to project by choosing "Default," "Always," "Only On Issues I Subscribe To," or "Never."
```

File: src/docs/product/alerts/create-alerts/metric-alert-config.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+The following set of filters translates into a [Discover query](/product/discover-queries/) that is displayed in the chart at the top of the alert configuration page.
+
+### Environment
+
+Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.
+
+The “Env:” dropdown list here has the same environments that are available for the selected project in the global “Environment” dropdown (this does not include hidden environments). Selecting "All" is equivalent to having no environment filter.
+
+### Event Type
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/metric-alert-config.mdx:21]:
```suggestion
### Event Type

```

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+  - Send a [Microsoft Teams](/product/integrations/msteams/) notification
+  - Send a notification to all legacy integrations.
+  - Send a notification using an [integration](/product/integrations/) built on the [Integration Platform](/product/integrations/integration-platform/)
+- Create an issue for an [integration](/product/integrations/), which includes:
+  - [Jira](/product/integrations/jira/)
+  - [Azure DevOps](/product/integrations/azure-devops)
+
+Learn more about [routing alerts with integrations](/product/sentry-basics/guides/alert-notifications/routing-alerts/).
+
+### Issue Owners
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:70]:
```suggestion
### Issue Owners

```

File: src/docs/product/alerts/alert-types.mdx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+1. [Triggers](/product/alerts/create-alerts/issue-alert-config/#when-conditions-triggers) ("When...")
+2. [Filters](/product/alerts/create-alerts/issue-alert-config/#if-conditions-filters) ("If...")
+3. [Actions](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) ("Then...").
+
+Triggers specify what type of activity you'd like monitored, or **when** an alert should be triggered.
+
+Filters help control noise by triggering an alert only **if** the issue matches the specified criteria.
+
+**Then**, actions specify what should happen when the trigger conditions are met and the filters match.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:36]:
```suggestion
1. [Triggers](/product/alerts/create-alerts/issue-alert-config/#when-conditions-triggers) specify what type of activity you'd like monitored, or **When** an alert should be triggered.
2. [Filters](/product/alerts/create-alerts/issue-alert-config/#if-conditions-filters) help control noise by triggering an alert only **If** the issue matches the specified criteria.
3. **Then**,[Actions](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions) specify what should happen when the trigger conditions are met and the filters match.
```

File: src/docs/product/alerts/alert-types.mdx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+<Note>
+
+Metric alerts aren’t affected by [ignored issues](/product/issues/states-triage/#ignore), so events from those issues are counted if they match the filters of your metric alert rule.
+
+</Note>
+
+### Alert Details
+
+The **Alert Details** page shows you the history of a metric alert rule for the last 24 hours by default, though can modify the display using the "Display" dropdown. When an alert is triggered, clicking the notification you receive takes you to this page, which displays the period when the alert was active. The page also includes details such as the alert rule conditions, the current status of the alert, and a summary of how much time the alert spent in each state (Critical, Warning, or Resolved).
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:67]:
```suggestion
The **Alert Details** page shows you the history of a metric alert rule for the last 24 hours by default, though can modify the time period using the "Display" dropdown. When an alert is triggered, clicking the notification you receive takes you to this page, which displays the period when the alert was active. The page also includes details such as the alert rule conditions, the current status of the alert, and a summary of how much time the alert spent in each state (Critical, Warning, or Resolved).
```

File: src/docs/product/alerts/create-alerts/index.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+     - Number of Errors
+     - Users Experiencing Errors
+     - Throughput
+     - Transaction Duration
+     - Apdex
+     - Failure Rate
+     - Largest Contentful Display
+     - First Input Delay
+     - Cumulative Layout Shift
+     - Custom Metric
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/index.mdx:29]:
```suggestion
1. Choose what you want to be alerted about. Selecting “Issues” creates an [issue alert](/product/alerts/alert-types/#issue-alerts), while selecting any other option creates a [metric alert](/product/alerts/alert-types/#metric-alerts).
    ![Choice between alerts about Errors, Performance, or a Custom Metric](create-new-alert-rule.png)
```

File: src/docs/product/alerts/alert-types.mdx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+  - /product/alerts-notifications/alerts/
+  - /product/alerts-notifications/metric-alerts/
+  - /product/alerts-notifications/issue-alerts/
+description: "Learn how to configure and manage Metric Alerts."
+---
+
+You can create two types of alerts:
+
+1. **Issue alerts**: Trigger when an issue (a grouped set of error events) matches a specific criteria.
+2. **Metric alerts**: Monitor macro-level metrics for both error events and transaction events.
```
Comment (imatwawana) [inline src/docs/product/alerts/alert-types.mdx:16]: @adhiraj Fiona suggested we make these definitions parallel in structure. I've put in a suggestion below. Does it work for you?

```suggestion
2. **Metric alerts**: Trigger when macro-level metrics for error or transaction events cross specific thresholds.
```

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+
+"When" conditions, or triggers, specify what type of activity you'd like monitored for the issue:
+
+- First seen
+- Changes state from `resolved` to `unresolved`
+- Changes state from `ignored` to `unresolved`
+- Seen more than a certain number of times in an interval
+- Seen by more than a certain number of unique users in an interval
+
+Triggers are optional. If you don’t select a trigger, the "When" conditions are considered met by default. That is, _all_ issues will meet this condition.
```
Comment (imatwawana) [inline src/docs/product/alerts/create-alerts/issue-alert-config.mdx:35]: @adhiraj I added a sentence above to further clarify what we mean by "the 'When' conditions are considered met by default". Just confirming, it's all _issues_ that we're saying meet the criteria, correct (vs events or events and issues)?

**Review by imatwawana — COMMENTED** [review]
Added questions for technical review

File: src/docs/product/alerts/create-alerts/issue-alert-config.mdx (hunk 9 above)
Comment (adhiraj) [reply src/docs/product/alerts/create-alerts/issue-alert-config.mdx:35]: Yes correct, the alert is evaluated each time an _event_ is received, and it's the _issue_ for that event that meets the criteria

**Pull request merged**